crosswalks/usgs-modelcatalog.csv

```diff
@@ -0,0 +1,74 @@
+Property,usgs-modelcatalog
+codeRepository,software.usgs_software_gitlab_release_urls / software.external_software_repository_urls
+programmingLanguage,programming_language
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,item_type
+applicationSubCategory,type_keywords
+downloadUrl,software.download_url
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version
+storageRequirements,
+supportingData,
+author,person.name when person_type=Author
+citation,publications.model_citations
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,last_update
+datePublished,release_date
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,science_keywords / other_keywords
+license,license
+producer,organization / external_organization_name
+provider,
+publisher,
+sponsor,
+version,version
+isAccessibleForFree,
+isPartOf,related_modelcatalog_assets
+hasPart,related_modelcatalog_assets
+position,
+description,description
+identifier,identifier.name / identifier.id / identifier.url / identifier.type
+name,name
+sameAs,
+url,other_links.usgs_website / other_links.weblinks
+relatedLink,other_links.weblinks
+givenName,
+familyName,
+email,person.email
+affiliation,person.organization
+identifier,person.orcid
+name,person.name
```

Comment:
```suggestion
email,email
affiliation,organization
identifier,orcid
name,name
```

usgs-modelcatalog's `person.email` maps to Codemeta's `author.email` (ditto for the other three), so there is no need to repeat `person.` for each of them.

Reply:
Is there any way to note that person can be of several types and we only want to map these if usgs-modelcatalog's person is of type=author? (as opposed to type=point of contact or type=maintainer)

Reply:
Not really, but `when person_type=Author` on the `author` line already does it.